src/frequenz/dispatch/_dispatcher.py

```diff
@@ -198,23 +198,33 @@ async def run():
         ```
     """
 
+    # pylint: disable-next=too-many-arguments
     def __init__(
         self,
         *,
         microgrid_id: int,
```

Comment:
Nothing to do with this PR but I wonder if there is a reason why you need to pass the `microgrid_id` when creating the client. For edge clients this makes sense, but for cloud clients? You need to create one client per microgrid? This feels weird.

Reply:
.. this is an edge client?!